gnmi_server/gnoi.go

```diff
@@ -94,151 +93,6 @@ func (srv *FileServer)  Get(req *gnoi_file_pb.GetRequest, stream gnoi_file_pb.Fi
-
-	_, err := authenticate(srv.config, ctx)
-	if err != nil {
-		return nil, err
-	}
-	log.V(1).Info("gNOI: Reboot")
-	log.V(1).Info("Request:", req)
-	log.V(1).Info("Reboot system now, delay is ignored...")
-	// TODO: Support GNOI reboot delay
-	// Delay in nanoseconds before issuing reboot.
-	// https://github.com/openconfig/gnoi/blob/master/system/system.proto#L102-L115
-	config_db_json, err := io.ReadFile(fileName)
-	if errors.Is(err, os.ErrNotExist) {
-		fileName = ""
-	}
-	err = RebootSystem(string(config_db_json))
-	if err != nil {
-		return nil, err
-	}
-	var resp gnoi_system_pb.RebootResponse
-	return &resp, nil
-}
-
-// TODO: Support GNOI RebootStatus
-func (srv *SystemServer) RebootStatus(ctx context.Context, req *gnoi_system_pb.RebootStatusRequest) (*gnoi_system_pb.RebootStatusResponse, error) {
-	_, err := authenticate(srv.config, ctx)
-	if err != nil {
-		return nil, err
-	}
-	log.V(1).Info("gNOI: RebootStatus")
-	return nil, status.Errorf(codes.Unimplemented, "")
-}
-
-// TODO: Support GNOI CancelReboot
-func (srv *SystemServer) CancelReboot(ctx context.Context, req *gnoi_system_pb.CancelRebootRequest) (*gnoi_system_pb.CancelRebootResponse, error) {
-	_, err := authenticate(srv.config, ctx)
-	if err != nil {
-		return nil, err
-	}
-	log.V(1).Info("gNOI: CancelReboot")
-	return nil, status.Errorf(codes.Unimplemented, "")
-}
-func (srv *SystemServer) Ping(req *gnoi_system_pb.PingRequest, rs gnoi_system_pb.System_PingServer) error {
-	ctx := rs.Context()
-	_, err := authenticate(srv.config, ctx)
-	if err != nil {
-		return err
-	}
-	log.V(1).Info("gNOI: Ping")
-	return status.Errorf(codes.Unimplemented, "")
-}
-func (srv *SystemServer) Traceroute(req *gnoi_system_pb.TracerouteRequest, rs gnoi_system_pb.System_TracerouteServer) error {
-	ctx := rs.Context()
-	_, err := authenticate(srv.config, ctx)
-	if err != nil {
-		return err
-	}
-	log.V(1).Info("gNOI: Traceroute")
-	return status.Errorf(codes.Unimplemented, "")
-}
-func (srv *SystemServer) SetPackage(rs gnoi_system_pb.System_SetPackageServer) error {
-	ctx := rs.Context()
-	_, err := authenticate(srv.config, ctx)
-	if err != nil {
-		return err
-	}
-	log.V(1).Info("gNOI: SetPackage")
-	return status.Errorf(codes.Unimplemented, "")
-}
-func (srv *SystemServer) SwitchControlProcessor(ctx context.Context, req *gnoi_system_pb.SwitchControlProcessorRequest) (*gnoi_system_pb.SwitchControlProcessorResponse, error) {
-	_, err := authenticate(srv.config, ctx)
-	if err != nil {
-		return nil, err
-	}
-	log.V(1).Info("gNOI: SwitchControlProcessor")
-	return nil, status.Errorf(codes.Unimplemented, "")
-}
-func (srv *SystemServer) Time(ctx context.Context, req *gnoi_system_pb.TimeRequest) (*gnoi_system_pb.TimeResponse, error) {
-	_, err := authenticate(srv.config, ctx)
-	if err != nil {
-		return nil, err
-	}
-	log.V(1).Info("gNOI: Time")
-	var tm gnoi_system_pb.TimeResponse
-	tm.Time = uint64(time.Now().UnixNano())
-	return &tm, nil
-}
-
```

Comment:
One problem with this cosmetic change here is that it is hard for reviewer to distinguish between cosmetic changes and non-cosmetic changes if they just see a huge trunk of code added in a new file. Then I might end up comment on existing code's issues rather than issues specific to the feature.

Reply:
Agreed, that makes sense. Will follow that next time!